pages/test/location_input.tsx

```diff
@@ -0,0 +1,28 @@
+import { useState } from 'react';
+import Button from '~/components/Button';
+import MapBoxInput from '~/components/mapbox_input';
+
+export default function Home() {
+  const [isSubmissionEnabled, setIsSubmissionEnabled] = useState(false);
+  const [zoomLevelThreshold, setZoomLevelThreshold] = useState(10);
```

Comment:
Don't really get why this is in a useState—seems like this is something we can just hardcode